pkg/cli-runtime/printer/tabwriter/tabwriter.go

```diff
@@ -95,7 +94,6 @@ type cell struct {
 // The Writer must buffer input internally, because proper spacing
 // of one line may depend on the cells in future lines. Clients must
 // call Flush when done calling Write.
```

Comment:
remove all newly added new lines in this PR

Reply:
Done